page.codeberg.dnkl.foot.yml

```diff
@@ -0,0 +1,83 @@
+app-id: page.codeberg.dnkl.foot
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: foot-wrapper.sh
+finish-args:
+  - --device=dri
+  - --socket=wayland
+  - --talk-name=org.freedesktop.Flatpak
+rename-desktop-file: foot.desktop
+rename-icon: foot
+cleanup:
+  - /etc
+  - /include
+  - /lib/pkgconfig
+  - /share/bash-completion
+  - /share/doc
+  - /share/fish
+  - /share/zsh
+modules:
+  # host-spawn is a tool required to run host terminal from flatpak sandbox
+  - name: host-spawn
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+        GOFLAGS: -modcacherw
+        GOPATH: /run/build/host-spawn
+    sources:
+      - type: git
+        url: https://github.com/1player/host-spawn
+        tag: 'v1.6.1'
+      - dest: vendor
+        path: modules.txt
+        type: file
+      - dest: vendor/github.com/godbus/dbus/v5
+        sha256: 03dfa8e71089a6f477310d15c4d3a036d82d028532881b50fee254358e782ad9
+        strip-components: 4
+        type: archive
+        url: https://proxy.golang.org/github.com/godbus/dbus/v5/@v/v5.1.0.zip
+      - dest: vendor/github.com/pkg/term
+        sha256: 943df71d3bc33501c59e581953fc455a6286c02a6dc92ffcf0be3aaab203fef9
+        strip-components: 3
+        type: archive
+        url: https://proxy.golang.org/github.com/pkg/term/@v/v1.1.0.zip
+      - dest: vendor/golang.org/x/sys
+        sha256: 8612eb416c739c3b04ce48dcbe65632c6fbc427031fd981caeceec6410d1e1fc
+        strip-components: 3
+        type: archive
+        url: https://proxy.golang.org/golang.org/x/sys/@v/v0.15.0.zip
+    build-commands:
+      - go install -mod=vendor
+  - name: tllist
+    builddir: true
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://codeberg.org/dnkl/tllist/archive/1.1.0.tar.gz
+        sha256: 0e7b7094a02550dd80b7243bcffc3671550b0f1d8ba625e4dff52517827d5d23
+  - name: fcft
+    builddir: true
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://codeberg.org/dnkl/fcft/archive/3.1.10.tar.gz
+        sha256: 4f4807d708f3a195e9a3caaa1ff9171b678af63a7af1c470a8984d601a4514de
+  - name: foot
+    builddir: true
+    buildsystem: meson
+    config-opts:
+      - -Dterminfo=disabled
+    build-commands:
+      - desktop-file-edit --set-key="Exec" --set-value="foot-wrapper.sh" "../foot.desktop"
+      - install -Dm755 ../foot-wrapper.sh -t /app/bin/
```

Comment:
this needs to be in `post-install` build-commands are provided by the `buildsystem: meson `